gatsby/content/blog/2023/01/2023-01-31-synapse-1.76.mdx

```diff
@@ -0,0 +1,85 @@
+---
+date: '2023-01-31'
+title: Synapse 1.76 released
+categories:
+- Releases
+author: H. Shay
+---
+
+Greetings matrix fans! We've published Synapse [version 1.76](https://github.com/matrix-org/synapse/releases/tag/v1.76.0)
+as the new stable release this week. Synapse admins are encouraged to upgrade
+to it at their convenience. Please take a look at the [upgrade notes](https://matrix-org.github.io/synapse/v1.76/upgrade.html#upgrading-to-v1760)
+for any important information about upgrading.
+
+## Announcements
+
+The big news is that faster joins for all has arrived! When joining a room for the first time,
+Synapse 1.76.0 will request a partial join from the other server by default. Previously, server admins
+had to opt-in to this using an experimental config flag.
+
+Server admins can opt out of this feature for the time being by setting
+
+```
+experimental:
+    faster_joins: false
+```
+
+in their server config.
+
+For everyone else, after a faster join Synapse considers that room "partially joined". In this state, you should be able to
+
+- read incoming messages;
```

Comment:
do/should we note that this only applies to clients which use lazyloaded members?

Reply:
I don't have a strong opinion on this, what do you think we should say here?

Reply:
I guess *most people* will be using Element, but on Matrix.org I like to think we try to be quite neutral so I think it would be worth pointing out the caveat. I'd probably write as 'read incoming messages (on clients which support lazy-loaded members in `/sync`)'

Reply:
Done, thanks for the suggestion and review!